scripts/us_bls/jolts/BLSJolts_StatisticalVariables.mcf

```diff
@@ -0,0 +1,1416 @@
+
+Node: dcid:jobChangeEvent
+name: "jobChangeEvent"
+typeOf: schema:Property
+domain: schema:Person
+range: dcs:JobChangeEventEnum
+
+Node: dcid:JobChangeEventEnum
+name: "JobChangeEventEnum"
+description: "Describes different job change events that may happen to workers" 
+typeOf: dcs:Class
+subClassOf: Enumeration
+
+Node: dcid:Hired
```

Comment:
Update this to Hire? I see you made the change in the StatVars, maybe missed the schema additions.

Reply:
Updated.